api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml

```diff
@@ -240,6 +240,83 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /ping:
```

Comment:
@helenosheaa Only review this one. Any corrections here, I'll paste to the others.